src/lib.rs

```diff
@@ -219,14 +229,23 @@ pub mod pallet {
 		///   deletion
 		#[transactional]
 		#[pallet::weight(T::WeightInfo::create_supersig())]
-		pub fn create_supersig(origin: OriginFor<T>, members: Vec<T::AccountId>) -> DispatchResult {
+		pub fn create_supersig(
+			origin: OriginFor<T>,
+			members: Vec<(T::AccountId, Roles)>,
+		) -> DispatchResult {
 			let who = ensure_signed(origin)?;
-			let supersig = Supersig::new(members.clone()).ok_or(Error::<T>::InvalidSupersig)?;
+			// let mut members = members.clone();
+			// members.sort();
+			// members.dedup();
+			let member_length = members.len();
+			if member_length < 2 {
+				return Err(Error::<T>::InvalidSupersig.into())
+			}
```

Comment:
a one member supersig mean we need 1 vote to validate a call.
If the sole member is a master, it mean his vote power is 0 (``1 / 2 = 0`` in rust). So the supersig will be locked, cant be modified, cant event be deleted.

So either a supersig of one user is only for a member and not a master, or a master have a 1 power vote if he is alone. But it is additional checks, so I'd like to ask @decentration if it could be a common use case, to know if it is worth to put those additional checks